WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -1110,6 +1113,59 @@ protected void onPostExecute(Spanned spanned) {
         }
     }
 
+    private String getUploadErrorHtml(String mediaId, String path) {
```

Comment:
This code duplication feels pretty risky, since we could easily change image containers in the future and forget to fix this too. What if we offloaded some of the work to the ZSSEditor? Add a special placeholder maybe (`[legacy-failed-image id=mediaId data-failed='try again message' path=path]`), and replace it with actual HTML in `ZSSEditor.init()` maybe?